sunpy/map/tests/test_map_factory.py

```diff
@@ -315,3 +315,21 @@ def test_map_fits():
     fits_map = sunpy.map.Map(AIA_171_IMAGE, memmap=True)
     assert isinstance(fits_map, sunpy.map.GenericMap)
     assert fits_map.data.base is not None
+
+def test_map_list_of_files():
+    files = [AIA_171_IMAGE, get_test_filepath('aia_lev1_211a_2022_03_30t17_21_21_63z_image_lev1.fits')]
+    with pytest.warns(SunpyUserWarning, match='Failed to read'):
+        amap = sunpy.map.Map(files, silence_errors=True)
+        assert amap.data.shape == (128,128)
+
+    files = [AIA_171_IMAGE, get_test_filepath('aia_lev1_211a_2022_03_30t17_21_21_63z_image_lev1.fits'), AIA_171_IMAGE]
```

Comment:
```suggestion
    files = [AIA_171_IMAGE, get_test_filepath('not_actually_fits.fits'), AIA_171_IMAGE]
```
When the file has been renamed.